### DISCUSSION TITLE
텍스트(폰트) 엑박 현상 관련

### DISCUSSION THREAD

**Discussion by JAICHANGPARK** [bosskmk/pluto_grid#593] [Ideas]
컬럼 사이즈를 조절하다보면 
엑박현상이 발생하곤 하는데 좋은 해결방안이 어디 없을까요? 

<img width="316" alt="image" src="https://user-images.githubusercontent.com/19484515/192923330-bef34c37-7e39-4988-82f1-8abb42aa42ba.png">

**Comment by bosskmk** [comment]
Flutter 웹의 캔버스 모드에서 발생하는 문제이고 아직 Flutter canvas 렌더링 엔진에서 해결 되지 않고 있는걸로 알고 있습니다.

flutter 를 데스크톱에서 웹으로 실행하면 자동으로 canvas 모드로 렌더링 되는데(모바일은 html)
이를 html 모드로 실행하게 설정하면 폰트 문제는 없는 것으로 알고 있습니다.

단 html 렌더링 엔진에 border 렌더링 문제가 있습니다.(stable 3.3.3)
때문에 PlutoGrid 의 각 행의 border 가 정상적으로 출력되지 않습니다(html 모드에서)
이 문제는 이슈를 제출했고 master 버전  3.4.0-28.0.pre.122 에서 해결 되었습니다.

현재 상태에서는 
Flutter Master 채널에서 PlutoGrid 를 html 렌더링 모드로 실행하면 문제 해결 되는 것으로 정리 됩니다.
Flutter Stable 채널에서 PlutoGrid 를 html 렌덜이 모드로 실행하는 것은 font 문제는 없을것이지만 행의 보더가 이상하게 출력 됩니다.
https://github.com/flutter/flutter/issues/111750 (해결 됨)

**Reply by JAICHANGPARK** [reply]
상세한 설명 감사합니다 🙇‍♂️
렌더러를 html로 설정하고 사용하는게 지금으로선 최고의 해결방법이겠군요..😵‍💫

**Comment by JAICHANGPARK** [comment]
https://jsfiddle.skia.org/canvaskit/3af3029a9799e5efbc7d46798219dcbca8664c344280312e946ac9499af0834e
